tests/test_snipping.py

```diff
@@ -104,9 +104,7 @@ def test_pileup(request):
             "end": [107_000_000, 113_000_000],
         }
     )
-    stack = cooltools.api.snipping.pileup(
-        clr, windows, view_df, exp, flank=None
-    )
+    stack = cooltools.api.snipping.pileup(clr, windows, view_df, exp, flank=None)
```

Comment:
speaking of PR-size - where are all this tiny formatting changes coming from ? black ?

I thought we ran black on the whole thing before ... should there be less changes like that going further. Is it the black's version changes that promote these changes ?

those are actually annoying a bit, I think - I wonder if we could try to separate formatting and substance-filled PRs from each other ?
It's not a big deal - but this happens a lot with different PRs and I though we could optimize it

Reply:
yes... I would ideally not have needed to run black, but pytest wouldn't run w/o it...

Reply:
or different black settings between people ... e.g. wrapping line at `80` chars or `100` ?... dunno

we should discuss that as a group - however minor it is - it would simplify PR reading, I think